fastn-core/tests/20-fastn-update-check/cmd.p1

```diff
@@ -0,0 +1,9 @@
+-- fbt:
+cmd: $FBT_CWD/../target/debug/fastn --test update --check
+exit-code: 1
+
+-- stdout:
+
+-- stderr:
+
+FastnCoreError(UpdateError { message: "Check error: Write operation detected during check for package 'fastn-community.github.io/bling' at '<cwd>/.packages/fastn-community.github.io/bling/Changelog.md'" })
```

Comment:
@harshdoesdev this is a very bad error message to show to end users.

Reply:
I have updated the error message